### PR TITLE
BB-1311: set default format for arithmetic m.

### PR DESCRIPTION
**Pull request by jkittler**
- for operation change is in percent

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-opensource-repository-check/136/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/162/ : SUCCESS in 4m 28s

File: src/DataLayer/converters/MeasureConverter.ts [hunk 1]
```diff
@@ -166,6 +166,12 @@ function getFormat(measure: IMeasure): string | undefined {
     const { measure: { definition } } = measure;
     const measureFormat = get(measure.measure, 'format');
 
+    if (VisualizationObject.isArithmeticMeasureDefinition(definition)) {
```
Comment (martinmosko) [inline src/DataLayer/converters/MeasureConverter.ts]: Please add also unit test which is covering this change :)

File: src/DataLayer/converters/MeasureConverter.ts [hunk 2]
```diff
@@ -166,6 +166,12 @@ function getFormat(measure: IMeasure): string | undefined {
...
+        if (definition.arithmeticMeasure.operator === 'change') {
+            return '#,##0.00%';
```
Comment (tomassatinsky) [inline src/DataLayer/converters/MeasureConverter.ts]: I don't know how's that possible, but it works also for localized number formatting

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-opensource-repository-check/137/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/163/ : FAILURE in 1m 00s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-opensource-repository-check/138/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/164/ : SUCCESS in 1m 11s

File: src/DataLayer/converters/tests/MeasureConverter.spec.ts [hunk 3]
```diff
@@ -114,4 +115,15 @@ describe('convertMeasure', () => {
...
         });
     });
+
+    describe('getFormat', () => {
+        it('should return default format for arithmetic measure sum operation', () => {
+            expect(MeasureConverter.getFormat(measures.arithmeticMeasure)).toBeUndefined();
+        });
+
+        it('should return percentage format for change operation', () => {
+            const changeAM = set(measures.arithmeticMeasure, 'measure.definition.arithmeticMeasure.operator', 'change');
```
Comment (martinmosko) [inline src/DataLayer/converters/tests/MeasureConverter.spec.ts]: please do not use `lodash/set` I would rather see mocked objects. :) 

reason is that lodash/set is not providing good type-safety... meaning if you change  arithmeticMeasure definition due to some refactor then `measure.definition.arithmeticMeasure.operator` will be not found.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-opensource-repository-check/139/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/165/ : FAILURE in 35s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-opensource-repository-check/140/ : SUCCESS in 18s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/166/ : SUCCESS in 1m 07s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-opensource-repository-check/142/ : SUCCESS in 11s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/168/ : SUCCESS in 4m 30s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-opensource-repository-check/143/ : SUCCESS in 16s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/169/ : SUCCESS in 2m 53s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-merge-compliance-check/45/ : FAILURE in 29s
- gooddata-js-unit-tests-zuul-docker-el7 : SKIPPED

**Comment by martinmosko** [comment]
ok to test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-opensource-repository-check/145/ : SUCCESS in 11s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/171/ : SUCCESS in 49s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-merge-compliance-check/46/ : SUCCESS in 22s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/172/ : SUCCESS in 37s

**Pull request merged**